scripts/src/java/org/oppia/android/scripts/todo/data/BUILD.bazel

```diff
@@ -0,0 +1,20 @@
+"""
+Libraries corresponding to parse open_issues.json file.
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_library")
+
+kt_jvm_library(
+    name = "data_lib",
```

Comment:
I suggest just calling this & the directory "model" and bundling all model-related files in here (i.e. both Issue.kt and Todo.kt).

Reply:
Done.